packages/common/src/types/semanticLayer.ts

```diff
@@ -72,6 +72,9 @@ export type SemanticLayerQuery = {
     timezone?: string;
     pivot?: SemanticLayerPivot;
     filters: SemanticLayerFilter[];
+    customMetrics?: (Pick<SemanticLayerField, 'name' | 'aggType'> & {
```

Comment:
This is the common interface for pivotQueries - it needs to support custom metrics